README.md

```diff
@@ -352,6 +352,75 @@ This is required for some SQL operations that cannot be run within a transaction
 
 -----------------------------------------------------
 
+#### `pgm.renameType( type_name, new_type_name )`
+
+> Rename a data type - [postgres docs](http://www.postgresql.org/docs/current/static/sql-altertype.html)
```

Comment:
All other comment verbs are in 3rd person (adds / sets), just this one (and the one below) is not (i.e. is without the trailing 's').